README.md

```diff
@@ -1,2 +1,75 @@
 jsk_model_tools [![Build Status](https://travis-ci.org/jsk-ros-pkg/jsk_model_tools.png?branch=master)](https://travis-ci.org/jsk-ros-pkg/jsk_model_tools)
 ===============
+
+# Model Convert
+
+## Convert from CAD manually
+(cf. PR of function to automatically convert: https://github.com/euslisp/jskeus/pull/248)
+
+- Convert1. export stl mesh with SolidEdge
+
+=> my_model.stl is generated.
+
+- Convert2. generate urdf
+generate urdf with following contents.
+（assume that my_model.stl is located at /home/leus/my_model.stl）
+```
+<robot name="my_model_name">
+  <link name="root">
+    <inertial>
+     <origin xyz="0 0 0.5" rpy="0 0 0"/>
+     <mass value="1"/>
+     <inertia ixx="100"  ixy="0"  ixz="0" iyy="100" iyz="0" izz="100" />
+    </inertial>
+    <visual>
+     <origin xyz="0 0 0" rpy="0 0 0" />
+     <geometry>
+       <mesh filename="file:///home/leus/my_model.stl" />
+     </geometry>
+    </visual>
+    <collision>
+     <origin xyz="0 0 0" rpy="0 0 0"/>
+     <geometry>
+       <mesh filename="file:///home/leus/my_model.stl" />
+     </geometry>
+    </collision>
+  </link>
+</robot>
+```
+=> my_model.urdf is generated.
```

Comment:
'generated' seems 'automatically generated', but this urdf is written manually, is it correct?
if so, verb 'generate' seems not proper for me.